entries/checked-selector.xml

```diff
@@ -7,7 +7,8 @@
   </signature>
   <desc>Matches all elements that are checked or selected.</desc>
   <longdesc>
-    <p>The <code>:checked</code> selector works for checkboxes, radio buttons, and select elements. For select elements only, use the <code>:selected</code> selector.</p>
+    <p>The <code>:checked</code> selector works for checkboxes, radio buttons, and "selected" options of select elements.</p>
```

Comment:
I'm certainly open for improvements, but I can't quite understand what you're suggesting. What do you propose for this sentence?

> The <code>:checked</code> selector works for checkboxes, radio buttons, and "selected" options of select elements.